### ISSUE TITLE
Too generic error when HMC does not have WS enabled

### ISSUE THREAD

**Issue by andy-maier** [zhmcclient/python-zhmcclient#296]
### Actual behavior

When the zhmcclient is used against a HMC that does not have its Web Services enabled, it reacts with an HTTPError with status 500 and reason 999, which is somewhat awkward to handle for users:

```
zhmcclient._exceptions.HTTPError: 500,999: Console Internal Error: \n
HTTP status code: 500\n
The server encountered an internal error that prevented it from fulfilling this request.\n
<pre>javax.servlet.ServletException: Web Services are not enabled.\n
	at com.ibm.hwmca.fw.api.ApiServlet.execute(ApiServlet.java:135)\n
	at com.ibm.hwmca.fw.api.ApiServlet.doGet(ApiServlet.java:109)\n
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:624)\n
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)\n
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)\n
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)\n
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:218)\n
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)\n
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)\n
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)\n
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)\n
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)\n
	at com.ibm.hwmca.fw.tomcat.ReleaseUserContextValve.invoke(ReleaseUserContextValve.java:52)\n
	at com.ibm.hwmca.fw.servlet.util.ApiGuiDiscriminatorValve.invoke(ApiGuiDiscriminatorValve.java:100)\n
	at com.ibm.hwmca.fw.tomcat.HmcRequestDumper.doFilter(HmcRequestDumper.java:166)\n
	at org.apache.catalina.filters.RequestDumperFilter.doFilter(RequestDumperFilter.java:203)\n
	at com.ibm.hwmca.fw.tomcat.HmcRequestDumper.invoke(HmcRequestDumper.java:143)\n
	at com.ibm.hwmca.fw.tomcat.UTF8RequestEncodingValve.invoke(UTF8RequestEncodingValve.java:94)\n
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:442)\n
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1082)\n
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:623)\n
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)\n
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1153)\n
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)\n
	at java.lang.Thread.run(Thread.java:785)\n
</pre>
[GET https://9.152.171.216:6794/api/version]
```

### Expected behavior

It would be nice if the zhmcclient could surface this quite common situation more programmer-friendly.

### Execution environment

* zhmcclient version: 0.13.0
* Operating system (type+version): RHEL 7.3
* HMC version: 2.13.1
* CPC version: 2.13.1

**Comment by andy-maier** [comment]
Addressed in PR #385.

**Comment by andy-maier** [comment]
Closing it because it has been addressed in 0.15.0
